dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManager.java

```diff
@@ -52,13 +54,8 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
-import javax.annotation.PreDestroy;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.InitializingBean;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
+import static org.apache.dolphinscheduler.common.Constants.REGISTRY_DOLPHINSCHEDULER_MASTERS;
+import static org.apache.dolphinscheduler.common.Constants.REGISTRY_DOLPHINSCHEDULER_WORKERS;
```

Comment:
This part of checkstyle is wrong. It is not clear that CI has not detected it, but it can be detected locally. It is recommended to modify it.

Reply:
Is my mistake. I didn't import the style check configuration `style/checkstyle.xml`.

I have solved all the files related to my changes. In addition, I see many codes with the same problems.


I think we can improve the documentation provided to developers, which is expected to improve the quality of subsequent code.

From this configuration `style/checkstyle.xml` > `ImportOrder`:

```
        <module name="ImportOrder">
            <property name="staticGroups" value="org.apache.dolphinscheduler,org.apache,java,javax,org,com"/>
            <property name="separatedStaticGroups" value="true"/>

            <property name="groups" value="org.apache.dolphinscheduler,org.apache,java,javax,org,com"/>
            <property name="ordered" value="true"/>
            <property name="separated" value="true"/>
            <property name="option" value="top"/>
            <property name="sortStaticImportsAlphabetically" value="true"/>
        </module>
```